lib/ops/ops.go

```diff
@@ -1901,3 +1901,11 @@ type Identity interface {
 	// GetAuthGateway returns auth gateway configuration
 	GetAuthGateway(SiteKey) (storage.AuthGateway, error)
 }
+
+// GetSiteReportRequest specifies the request to get the site report
+type GetSiteReportRequest struct {
```

Comment:
Nit: I would call it `GetClusterReportRequest`. "Site" is an older term (despite still being used a lot in the code due to legacy reasons), we use "Cluster" everywhere now.